app/pandas.ipynb

```diff
@@ -208,18 +414,92 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
-   "outputs": [],
    "source": [
-    "# Write your code here"
+    "df[\"Average age\"] = df[\"Age\"].mean()\n",
```

Comment:
The 'Average age' column is being added with the same value for all rows, which is the mean of the 'Age' column. If the intention was to have a single value representing the average age of all entries, it might be better to store this as a separate variable or display it separately rather than adding it as a column with repeated values.